src/tic_tac_toe/io.clj

```diff
@@ -1,15 +1,19 @@
 (ns tic-tac-toe.io)
 
 (defprotocol IO
-  (user-input [x])
-  (display [x message])
-  (clear-io [x]))
+  (user-input [this])
+  (display [this message])
+  (clear-io [this])
+  (flush-io [this])
+  (pause-io [this]))
```

Comment:
Does this fit here? What does it mean to pause the io? Is this truly a responsibility of the IO?

Also, `-io` seems to be redundant since the name of the protocol is `IO`

Reply:
Made [pause](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-52c69c82bcdbcc16e3156d4a9033a37cR30) a function of the ui only.